programs/programs/co/__init__.py

```diff
@@ -63,5 +64,6 @@
     "dtr": DenverTrashRebate,
     "dptr": DenverPropertyTaxRelief,
     "nf": NurturingFutures,
+    "denver_sidewalk_rebate": DenverSidewalkRebate,
```

Comment:
What do you think about making the name `dsr` in line with how we name the other programs?
```suggestion
    "dsr": DenverSidewalkRebate,
```